libs/experimental/tests/unit_tests/test_data_anonymizer.py

```diff
@@ -0,0 +1,70 @@
+from typing import Dict, List
+
+import pytest
+from presidio_analyzer import PatternRecognizer
+from presidio_anonymizer.entities import OperatorConfig
+
+from langchain_experimental.data_anonymizer import PresidioAnonymizer
+
+
+@pytest.mark.parametrize(
```

Comment:
you need to mark it to be enabled if presidio is installed. You can take a peek on my docker containers tests

Reply:
Probably check should be for faker as well if it is used